packages/e2e/src/seeds/main.ts

```diff
@@ -20,15 +21,9 @@ export const seedData = async (specFileName: string) => {
   }
 }
 
-// TODO [mw] implement it - scope of the next PR
-const getData = (specFileName: string): SeedParams | undefined => {
-  return
-}
-
 export const removeSeededData = async () => {
-  console.log("Removing seed data result", seedResult)
-
   if (seedResult) {
+    console.log("Removing seed data result", seedResult)
```

Comment:
```suggestion

```

Reply:
Console log was added intentionally - might help in debugging if seeding would not work fine. I would opt for leaving it.